Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 Bugfixes:
  * Commandline Interface: Disallow the following options outside of the compiler mode: ``--via-ir``,``--metadata-literal``, ``--metadata-hash``, ``--model-checker-show-unproved``, ``--model-checker-div-mod-no-slacks``, ``--model-checker-engine``, ``--model-checker-invariants``, ``--model-checker-solvers``, ``--model-checker-timeout``, ``--model-checker-contracts``, ``--model-checker-targets``.
  * Type Checker: Fix null dereference in `abi.encodeCall` type checking of free function.
+ * Type Checker: Fix compiler crashing because of nested tuple assignment.
```

Comment:
```suggestion
 * Type Checker: Fix compiler crash on tuple assignments involving certain patterns with unary tuples on the left-hand side.
```
Don't want to be pedantic, but just so we can be more specific and precise here.
As discussed in the team chat and suggested by @ekpyron.
Also, a rebase on develop should fix external tests and then PR is good to go.

Reply:
Haha don't worry. I do prefer the new changelog description.

Fixed and rebased!